jest-runner-eslint.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+    cliOptions: {
+        config: '.eslintrc.json',
+        ext: ['.ts', '.js'],
+    },
```

Comment:
can we add the cache option back in here? might not make a difference now but it might in the future